.travis.yml

```diff
@@ -19,16 +25,20 @@ branches:
 notifications:
   email: false
 
+codecov: true
+
 jobs:
   include:
     - stage: "Documentation"
-      julia: 1.1
+      julia: 1.3
       os: linux
       script:
         - julia --project=docs/ -e 'using Pkg; Pkg.instantiate(); Pkg.build()'
         - julia --project=docs/ docs/make.jl
       name: "HTML"
       after_success: skip
-
-after_success:
-  - julia -e 'import Pkg; Pkg.add("Coverage"); using Coverage; Codecov.submit(Codecov.process_folder())'
+  exclude:
+  - os: osx
+    arch: x86
+  allow_failures:
```

Comment:
This should be removed imo. We want this package to work on nightly and some red crosses might make someone annoyed enough to look into it.